### PR TITLE
Add Pod creation example in k8s.io/docs/concepts/workloads/pods/

### PR DESCRIPTION
**Pull request by 100mik**
Adding example requested in #28848

### CODE REVIEW

File: content/en/docs/concepts/workloads/pods/_index.md [hunk 1]
```diff
@@ -46,6 +46,32 @@ further sub-isolations applied.
...
+    image: nginx:1.14.2
+    ports:
+    - containerPort: 80
+```
+
+To create the Pod shown above, run the following command:
+```shell
+kubectl apply -f https://k8s.io/examples/pods/simple-pod.yaml
+```
+
```
Comment (100mik) [inline content/en/docs/concepts/workloads/pods/_index.md:70]: Do we want to add steps to list Pods and describe what the user sees as we have in the [deployments documentation](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#creating-a-deployment)?

File: content/en/docs/concepts/workloads/pods/_index.md [hunk 2]
```diff
@@ -46,6 +46,32 @@ further sub-isolations applied.
...
+Pods are generally not created directly and are created using workload resources.
+See [Working with Pods](#working-with-pods) for more information on how Pods are used
+with workload resources.
```
Comment (100mik) [inline content/en/docs/concepts/workloads/pods/_index.md:73]: Should we remove lines 71-73 as the section right after it has similar information in more detail?

File: content/en/docs/concepts/workloads/pods/_index.md [hunk 3]
```diff
@@ -46,6 +46,32 @@ further sub-isolations applied.
...
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx:1.14.2
+    ports:
+    - containerPort: 80
+```
```
Comment (sftim) [inline content/en/docs/concepts/workloads/pods/_index.md:64]: You can use a special shortcode for this:
```suggestion
{{< codenew file="pods/simple-pod.yaml" >}}
```

File: content/en/docs/concepts/workloads/pods/_index.md [hunk 4]
```diff
@@ -46,6 +46,32 @@ further sub-isolations applied.
 In terms of Docker concepts, a Pod is similar to a group of Docker containers
 with shared namespaces and shared filesystem volumes.
 
+## Creating a Pod
```
Comment (sftim) [inline content/en/docs/concepts/workloads/pods/_index.md:49]: I'd have the “Using Pods” heading here.

File: content/en/docs/concepts/workloads/pods/_index.md [hunk 5]
```diff
@@ -46,6 +46,32 @@ further sub-isolations applied.
...
+To create the Pod shown above, run the following command:
+```shell
+kubectl apply -f https://k8s.io/examples/pods/simple-pod.yaml
+```
+
+Pods are generally not created directly and are created using workload resources.
+See [Working with Pods](#working-with-pods) for more information on how Pods are used
+with workload resources.
+
 ## Using Pods
```
Comment (sftim) [inline content/en/docs/concepts/workloads/pods/_index.md:75]: This heading could become:
```markdown
### Workload resources for managing pods
```

**Review by sftim — COMMENTED** [review]
We should definitely switch to the `codenew` shortcode; other feedback is nits (stuff that doesn't block a merge).

File: content/en/docs/concepts/workloads/pods/_index.md (hunk 3 above)
Comment (100mik) [reply content/en/docs/concepts/workloads/pods/_index.md]: That's really cool! Amended the commit to resolve this and the nits

**Review by sftim — COMMENTED** [review]
There's more to clear up on that page; this is a nice start.
/lgtm
/approve

**Pull request merged**